src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

Comment:
The color value for the box-shadow should be stored in a variable for better maintainability and to comply with the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Consider using a variable like `$box-shadow-color` in `src/styles/utils/variables.scss` and then use it here.